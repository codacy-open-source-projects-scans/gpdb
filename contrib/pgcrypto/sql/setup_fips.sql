-- Setup for fips.sql test

-- We are setting shared_preload_libraries so that we can set the extension GUC
-- 'pgcrypto.fips' immediately after creating the extension on master.

-- start_ignore
\! gpconfig -c shared_preload_libraries -v "$(psql -At -c "SELECT array_to_string(array_append(string_to_array(current_setting('shared_preload_libraries'), ','), 'pgcrypto.so'), ',')" postgres)"
\! gpstop -air
-- end_ignore
