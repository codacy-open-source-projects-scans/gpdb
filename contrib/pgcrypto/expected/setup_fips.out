-- Setup for fips.sql test
-- We are setting shared_preload_libraries so that we can set the extension GUC
-- 'pgcrypto.fips' immediately after creating the extension on master.
-- start_ignore
\! gpconfig -c shared_preload_libraries -v '$libdir/pgcrypto'
\! gpstop -air
-- end_ignore
